src/HttpWorker.php

```diff
@@ -252,7 +252,10 @@ private function arrayToHeaderValue(array $headers = []): array
          * @var array<array-key, string> $value
          */
         foreach ($headers as $key => $value) {
-            $result[$key] = new HeaderValue(['value' => $value]);
+            $value = \array_filter((array) $value, static fn (mixed $v): bool => \is_string($v));
+            if ($value !== []) {
+                $result[$key] = new HeaderValue(['value' => $value]);
+            }
```

Comment:
```suggestion
            $value = \array_filter(
                \is_array($value) ? $value : [$value],
                static fn (mixed $v): bool => \is_string($v),
            );
            if ($value !== []) {
                $result[$key] = (new HeaderValue())->setValue($value);
            }
```

Might $value be object? We can't just cast it into array then.

Reply:
Thanks!